Sources/Classes/RSMoEngageDestination.swift

```diff
@@ -137,15 +165,15 @@ extension RSMoEngageDestination {
         for (key, value) in traits {
             if value is String || value is NSNumber || value is Date {
                 switch key {
-                case RSKeys.Identify.Traits.email: MoEngage.sharedInstance().setUserEmailID(value as? String)
-                case RSKeys.Identify.Traits.name: MoEngage.sharedInstance().setUserName(value as? String)
-                case RSKeys.Identify.Traits.phone: MoEngage.sharedInstance().setUserMobileNo(value)
-                case RSKeys.Identify.Traits.firstName: MoEngage.sharedInstance().setUserAttribute(value, forKey: USER_ATTRIBUTE_USER_FIRST_NAME)
-                case RSKeys.Identify.Traits.lastName: MoEngage.sharedInstance().setUserLastName(value as? String)
-                case RSKeys.Identify.Traits.gender: MoEngage.sharedInstance().setUserAttribute(value, forKey: USER_ATTRIBUTE_USER_GENDER)
-                case RSKeys.Identify.Traits.birthday: identifyDateUserAttribute(value: value, key: USER_ATTRIBUTE_USER_BDAY)
-                case RSKeys.Identify.Traits.address: MoEngage.sharedInstance().setUserAttribute(value, forKey: RSKeys.Identify.Traits.address)
-                case RSKeys.Identify.Traits.age: MoEngage.sharedInstance().setUserAttribute(value, forKey: RSKeys.Identify.Traits.age)
+                case RSKeys.Identify.Traits.email: MoEngageSDKAnalytics.sharedInstance.setEmailID((value as? String)!)
+                case RSKeys.Identify.Traits.name: MoEngageSDKAnalytics.sharedInstance.setName((value as? String)!)
+                case RSKeys.Identify.Traits.phone: MoEngageSDKAnalytics.sharedInstance.setMobileNumber(value as! String)
+                case RSKeys.Identify.Traits.firstName: MoEngageSDKAnalytics.sharedInstance.setUserAttribute(value, withAttributeName: "USER_ATTRIBUTE_USER_FIRST_NAME") //USER_ATTRIBUTE_USER_FIRST_NAME
+                case RSKeys.Identify.Traits.lastName: MoEngageSDKAnalytics.sharedInstance.setLastName((value as? String)!)
```

Comment:
We should not abort the execution if the value is not present. This might crash the SDK. Please make the necessary changes and remove the use of `!`.

Reply:
done